lint/ilib-lint-config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "lint-test",
+    "plugins": [
+        "python",
+        "python-gnu"
+    ],
+    "locales": [
+        "en-US",
+        "de-DE"
+    ],
+    "filetypes": {
+        "xliff": {
+            "ruleset": [ "python", "python-gnu" ]
```

Comment:
xliff filetype apply python and python-gun ruleset?
How about the *.py file? there is src/prime.py file here.If python plugins are written in the config file, no need to write additional file type or path info regarding *.py file? In case enough follow plugins' default info?

I haven't executed this sample yet. so I may not clearly understand well..

Reply:
If the resources for your project come from python files, then python rules should be applied to strings in the xliff files. This will check for things like python-style replacement parameter syntax. Other general rules can be applied to the resources as well and the ruleset array can contain other ruleset names.

The python plugin doesn't yet include a parser or rules to check source code. Currently, there are only rules in the plugin to check strings in resource files. In fact, the ability to read any regular source code and apply rules to it hasn't been built into the ilib-lint tool yet. (That's a feature coming up soon!) Also, the loctool python plugin is not finished yet, so we cannot run the loctool to automatically extract the strings from the python source code and put them into an xliff file.

For this sample, we need to have the python plugins approved and published to npm first before you can try it out yourself.